content/en/blog/_posts/2024-08-20-websocket-transition.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: blog
+title: 'Streaming Transitions from SPDY to WebSockets'
+date: 2024-08-20
+slug: websockets-transition
+author: >
+  [Sean Sullivan](https://github.com/seans3) (Google)
+  [Shannon Kularathna](https://github.com/shannonxtreme) (Google)
+---
+
+In Kubernetes 1.31, by default kubectl now uses the WebSocket protocol
+instead of SPDY for streaming.
```

Comment:
```suggestion
In Kubernetes 1.31 and later, Kubernetes uses the WebSocket protocol instead of
SPDY when upgrading HTTP connections from `kubectl` to streaming connections.
```

Reply:
"connections from kubectl" might be inaccurate

Reply:
For me (and I'm an editor for the blog), the original wording is OK; this isn't documentation so being more conversational is OK.